.gitignore

```diff
@@ -1,6 +1,10 @@
 # vim
```

Comment:
gitignore = pork barrel :P If you don't like these entries, no worries :) I put the gitconfig changes in it's own commit, so I can revert it if need be pretty easy.